### ISSUE TITLE
Swap the origin of trips when appointment time is changed

### ISSUE THREAD

**Issue by davidtran** [clowdfish/ttg-frontend#120]
Hi @GrosR, I found a problem with the arrangement of trips.
For example, I have 3 appointments: 

Appointment A at 16-8
Appointment B at 17-8
Appointment C at 18-8

So my routes would be:
Start from X, and go to A
Start from A, I go to B
Start from B, I go to C...

Now if user changes the time of B to 18-8, and appointment C to 17-8. We need to change the origin of B & C too.

Now, 
Start from A, he goes to C
Start from C, he goes to B

If we don't do anything, the origin of C is still B and origin of B is still A

We need to apply this method to the home screen because we are auto sorting the appointments based on time.

**Comment by GrosR** [comment]
yes, I agree. Still, a user can change things in the select itinerary section, too. This is why I wrote in one of the latest tickets that we must ask the user how to proceed with these changes. I have to look up the ticket where I wrote that ... but it had to do with changes in the select itinerary section and with return flights. 

We must apply these rules to ensure that the trip will make sense. Still, we must inform the user that his change affects other appointments, too.

We should therefore show the user which appoinments are affected and let the user confirm, that we we swap origins / destinations or that the user can discard the appointment change. 

I will upate this issue once I found my latest comment... :-)
